Userland/Libraries/LibWeb/WebAssembly/WebAssemblyObject.cpp

```diff
@@ -399,8 +416,22 @@ Optional<Wasm::Value> to_webassembly_value(JS::Value value, const Wasm::ValueTyp
         return Wasm::Value { static_cast<float>(number) };
     }
     case Wasm::ValueType::FunctionReference:
+    case Wasm::ValueType::NullFunctionReference: {
+        if (value.is_null())
+            return Wasm::Value { Wasm::ValueType(Wasm::ValueType::NullExternReference), 0ull };
+
+        if (value.is_function()) {
+            auto& fn = value.as_function();
```

Comment:
This is not the serenity way

```suggestion
            auto& function = value.as_function();
```